go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/hashicorp/vault/api v1.9.2
 	github.com/jaegertracing/jaeger v1.41.0
 	github.com/knadh/koanf v1.5.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/connector/spanmetricsconnector v0.80.0
```

Comment:
To prevent moving the deps forward, do you mind using 0.79.0 for now? We have an automated job that will address them.

Reply:
Can do!